front/plugins/avahi_scan/avahi_scan.py

```
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+# !/usr/bin/env python3
```

Ruff (0.14.5)

The file is executable but no shebang is present

(EXE002)